site/src/pages/LoginPage/LoginPage.tsx

```diff
@@ -61,8 +54,8 @@ export const LoginPage: React.FC = () => {
               authMethods={authState.context.methods}
               redirectTo={redirectTo}
               isLoading={isLoading}
-              authErrorMessage={authErrorMessage}
-              methodsErrorMessage={getMethodsError}
+              authError={authState.context.authError}
+              methodsError={authState.context.getMethodsError as Error}
```

Comment:
Why do we have to override the compiler here?

Reply:
`getMethodsError` does not really throw an error from the backend, but only a generic one. This was being done earlier (in one of the lines we deleted), so just wanted to keep it the same for sanity (like referencing the message string from the Error object). 

Additionally, I don't know a good way to produce a `getMethodsError`, so couldn't reeally play around with it. @kylecarbs do you have any suggestions about how the error could be reproduced?

Reply:
I don't really think we do have to override it. I'm guessing that ended up in there originally to make it so we didn't have to add `| unknown` in a component prop type. I had hoped we could narrow all the error types in the XServices but it turns out we want to leave them alone until they reach `ErrorSummary` and `getFormHelpers`, so I think we're just stuck with `Error | unknown` and we can delete this. I like the comment explaining that this call never returns an ApiError though.